dask/base.py

```diff
@@ -568,7 +568,7 @@ def compute(
 
     Examples
     --------
-    >>> import dask as d
+    >>> import dask
```

Comment:
Looks like there is a `>>> d.compute(a, b)` line below that needs to change to `>>> dask.compute(a, b)`